src/libclipper/include/clipper/redis.hpp

```diff
@@ -118,6 +118,8 @@ boost::optional<std::string> get_current_model_version(
  *
  * \param container_name should be the name of a Docker container
  * \param model_data_path should be the path on the Clipper host
+ * \param designated_batch_size should be the user defined batch size of the container, it's default value(-1)
```

Comment:
Change the description to "should be the user-defined batch size for all replicas of the model. Its default value (-1) indicates that the batch size was unspecified and that Clipper should calculate it adaptively.